operationsgateway_api/src/records/echo_interface.py

```diff
@@ -81,6 +81,30 @@ def format_record_id(record_id: str, use_subdirectories: bool = True) -> str:
         else:
             return record_id
 
+    def put_lifecycle(self) -> None:
+        """
+        https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/put_bucket_lifecycle_configuration.html
+        Puts the lifecycle configuration of the bucket to delete objects after a certain
+        number of days.
+        """
+        if Config.config.echo.expiry_days is not None:
+            log.info("Put expiry lifecycle of %s days", Config.config.echo.expiry_days)
+            rule = {
+                "Expiration": {"Days": Config.config.echo.expiry_days},
+                "ID": "expiry",
+                "Prefix": "",
+                "Status": "Enabled",
+            }
+            try:
+                self.client.put_bucket_lifecycle_configuration(
+                    Bucket=Config.config.echo.bucket_name,
+                    LifecycleConfiguration={"Rules": [rule]},
+                )
+            except ClientError:
+                log.exception("Failed to put lifecyle")
```

Comment:
We should probably log the bucket name here, and anywhere else has tries to set / delete bucket policy